core/src/integTest/scala/water/sparkling/itest/local/H2OContextConversionOnSubsetExecutors.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package water.sparkling.itest.local
+
+import org.apache.spark.SparkContext
+import org.apache.spark.h2o.DoubleHolder
+import org.apache.spark.h2o.backends.internal.InternalBackendConf
+import org.apache.spark.h2o.utils.SharedH2OTestContext
+import org.apache.spark.rdd.RDD
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{BeforeAndAfter, FunSuite, Matchers}
+import water.fvec.{H2OFrame, Vec}
+
+
+@RunWith(classOf[JUnitRunner])
+class H2OContextConversionOnSubsetExecutors extends FunSuite
+  with Matchers with BeforeAndAfter with SharedH2OTestContext {
+
+  override def createSparkContext: SparkContext = new SparkContext(
+    "local-cluster[3,1,1024]",
+    this.getClass.getName,
+    conf = defaultSparkConf.set(InternalBackendConf.PROP_CLUSTER_SIZE._1, 1.toString))
+
+  test("asH2OFrame conversion on subset of executors") {
+    assert(hc.getH2ONodes().length == 1)
+    assert(spark.sparkContext.statusTracker.getExecutorInfos.length == 4)
```

Comment:
In case of local-cluster, the MASTER is for example `local-cluster[3,1,1024]` it means 3 worker nodes and 1 driver, so spark actually has 4 nodes. In case of local-cluster the `spark.sparkContext.statusTracker.getExecutorInfos` keeps count of the driver node as well

Reply:
Added comment to the code